### ISSUE TITLE
Zefyr 0.11 not compatible with flutter 1.22

### ISSUE THREAD

**Issue by robert-virkus** [memspace/zefyr#415]
Compile for Zefyr fails on Flutter 1.22

## Steps to Reproduce

1. Upgrade to flutter 1.22.0 (current stable)
2. Try to compile app with Zefyr 0.11

## Logs

```
../../flutter/.pub-cache/hosted/pub.dartlang.org/zefyr-0.11.0/lib/src/widgets/input.dart:11:7: Error: The non-abstract class 'InputConnectionController' is missing implementations for these members:
 - TextInputClient.currentAutofillScope
 - TextInputClient.performPrivateCommand
 - TextInputClient.showAutocorrectionPromptRect
```

```
PS C:\Users\User\Development\enough_mail_app> flutter doctor -v
[√] Flutter (Channel stable, 1.22.0, on Microsoft Windows [Version 10.0.19041.508], locale de-DE)
    • Flutter version 1.22.0 at C:\Users\User\flutter
    • Framework revision d408d302e2 (3 days ago), 2020-09-29 11:49:17 -0700
    • Engine revision 5babba6c4d
    • Dart version 2.10.0


[√] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at C:\Users\User\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Android Studio (version 4.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 49.0.2
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code (version 1.49.2)
    • VS Code at C:\Users\User\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.15.0

[√] Connected device (1 available)
    • sdk gphone x86 (mobile) • emulator-5554 • android-x86 • Android 11 (API 30) (emulator)

• No issues found!
```

**Comment by rodrigueaf** [comment]
Any workaround for this issue?

**Comment by pulyaevskiy** [comment]
I just published 0.12 which fixes this. Note that I'm not planning to release any new updates in the 0.x branch. Currently active development is focused in 1.0-dev branch, you can find details in: https://github.com/memspace/zefyr/issues/409
